include/aikido/planner/vectorfield/MoveEndEffectorPoseVectorField.hpp

```diff
@@ -0,0 +1,78 @@
+#ifndef AIKIDO_PLANNER_VECTORFIELD_MOVEENDEFFECTORPOSEVECTORFIELD_HPP_
+#define AIKIDO_PLANNER_VECTORFIELD_MOVEENDEFFECTORPOSEVECTORFIELD_HPP_
+
+#include <aikido/planner/vectorfield/BodyNodePoseVectorField.hpp>
+
+namespace aikido {
+namespace planner {
+namespace vectorfield {
+
+/// Vector field for moving end-effector to a goal pose in SE(3).
+/// It defines a vector field in meta-skeleton state space that moves
+/// an end-effector a desired pose in SE(3) by following a geodesic
+/// loss function in SE(3) via an optimized Jacobian.
+/// The geodesic loss function is defined as the geodesic (shortest
+///  path) from the current pose to the goal pose.
+///
+/// This class defines two callback functions for vectorfield planner.
+/// One for generating joint velocity in MetaSkeleton state space,
+/// and one for determining vectorfield planner status.
+class MoveEndEffectorPoseVectorField : public BodyNodePoseVectorField
+{
+public:
+  /// Constructor.
+  ///
+  /// \param[in] stateSpace MetaSkeleton state space.
+  /// \param[in] bn Body node of end-effector.
+  /// \param[in] goalPose Desired end-effector pose.
+  /// \param[in] poseErrorTolerance Constraint error tolerance in meters.
+  /// \param[in] r Conversion of radius to meters in computing Geodesic
+  /// distance.
+  /// \param[in] linearVelocityGain Linear velocity gain in workspace.
+  /// \param[in] angularVelocityGain Angular velocity gain in workspace.
+  /// \param[in] initialStepSize Initial step size.
```

Comment:
Clarify that this is the *maximum* step size used to guarantee that the integrator does not step out of joint limits, not the initial step size.